### PR TITLE
AC-565: Replace anonymous types with Java 8 lamdas.

### PR DESCRIPTION
**Pull request by Abdelaty**
## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-565

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [ ] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by f4ww4z** [comment]
@Abdelaty Please resolve the merge conflict.

**Comment by Abdelaty** [comment]
Done

**Comment by f4ww4z** [comment]
@Abdelaty Did you see [this error](https://travis-ci.org/openmrs/openmrs-contrib-android-client/builds/524965015#L2141) from failed CI build? Make sure the variable name is the same.

**Comment by Abdelaty** [comment]
Ok, thank Fawwaz.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsRecyclerViewAdapter.java [hunk 1]
```diff
@@ -52,20 +51,11 @@ public SettingsViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int view
...
-        if (position == 0) {
+        if (position==0){
             holder.mRowLayout.setOnClickListener(view -> {
-                Intent i = new Intent(view.getContext(), LogsActivity.class);
+                Intent i = new Intent(view.getContext() , LogsActivity.class);
                 view.getContext().startActivity(i);
-            });
-        }
-
-        if (position == 2) {
```
Comment (deepak140596) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsRecyclerViewAdapter.java]: @f4ww4z @Abdelaty , deletion of this line has led to a bug where clicking on Privacy Policy on the Settings page doesn't perform any action.

Comment (f4ww4z) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsRecyclerViewAdapter.java]: @deepak140596 I've assigned the issue to you. Good find.

**Pull request merged**